### PR TITLE
Chore: Addons can define their openpype alias name

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Changelog Description
Addon can be registered under 2 names in `AddonsManager`.

## Additional info
This is needed for `sitesync` addon which was named `sync_server` in OpenPype. With this change it is possible to do both:
```python
from ayon_core.addon import AddonsManager

manager = AddonsManager()
sync_server = manager["sync_server"]
sitesync = manager["sitesync"]
print(sync_server is sitesync)
>>> True
```

### CODE REVIEW

File: client/ayon_core/addon/base.py [hunk 1]
```diff
@@ -822,6 +830,17 @@ def initialize_addons(self):
                     exc_info=True
                 )
 
+        for item in aliased_names:
+            name_alias, addon = item
```
Comment (BigRoy) [inline client/ayon_core/addon/base.py:834]:
```suggestion
        for name_alias, addon in aliased_names:
```

Comment (iLLiCiTiT) [reply client/ayon_core/addon/base.py:834]: Some pythons have issue with this expanding in for loop.

Comment (BigRoy) [reply client/ayon_core/addon/base.py:834]: Which python? 

It works fine for me in Py 2.7, Py 3.7, Py 3.9 and Py 3.10 with the next code snippet:
```python
data = [(1, 2), (1001, 1010)]
for start, end in data:
    print(start, end)
```

Comment (iLLiCiTiT) [reply client/ayon_core/addon/base.py:834]: To be honest I don't remember, but I'm scared to use this now as we need to make release today.

**Review by kalisp — APPROVED** [review]
Site Sync working.

**Pull request merged**